spec/features/help_pages_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+describe "Help Pages", feature: true do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.